bundles/AdminBundle/Controller/Admin/UserController.php

```diff
@@ -1238,8 +1238,10 @@ protected function getUserId(Request $request)
     private function generateCustomUrl(array $params, $fallbackUrl = 'pimcore_admin_login_check', $referenceType = UrlGeneratorInterface::ABSOLUTE_URL): string
     {
         try {
+            $adminEntryPointRoute = $this->getParameter('pimcore_admin.custom_admin_route_name');
+
             //try to generate invitation link for custom admin point
-            $loginUrl = $this->generateUrl('my_custom_admin_entry_point', $params, $referenceType);
+            $loginUrl = $this->generateUrl($adminEntryPointRoute, $params, $referenceType);
```

Comment:
this will be a BC break if you have already defined a route with `my_custom_admin_entry_point`

Reply:
I would use `my_custom_admin_entry_point` as default value for BC reasons